Makefile

```diff
@@ -93,4 +93,7 @@ e2e-etcdbackuprecovery:
 e2e-keyrotation:
 	FOCUS="\[KeyRotation\]\[Fake\]" TIMEOUT=70m ./hack/e2e.sh
 
+e2e-scaleupdown:
+	FOCUS="\[ScaleUpDown\]\[Real\]" TIMEOUT=30m ./hack/e2e.sh
```

Comment:
Why use the real vs the fake RP by default? Partial requests do not work for the prod RP because my merge code is still not vendored.